ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml

```diff
@@ -159,12 +159,13 @@ tests:
       cli: latest
       commands: |
         export QUAY_PASS=$(cat /maistra-secrets/password)
+        oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge $$ sleep 20 
```

Comment:
did you mean `&&` instead of `$$`?

do we really need to sleep 20s?

Reply:
You can also remove the QUAY_PASS definition and the `credentials` section, as it's only used to retrieve the quay password...

Reply:
Done

Reply:
Fixed the $$ signs and decreased the sleep time, the issue here is that we need to wait until the route is created, so we need a minimum amount of time to let the route be created and be able to get DOCKER_INSECURE_REGISTRIES later. I decreased to 5 seconds

Reply:
Or we can add some type of validation to wait until the route is created, but will add more complexity to the code